python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -332,7 +347,10 @@ def finalize_dosage_scores_after_c_t(
     genos_adjusted = merged_allele_comparison_genos.apply(adjust_dosages, axis=1)
     genotypes_adjusted_only = genos_adjusted.iloc[:, 1:]
     genos_transpose = genotypes_adjusted_only.T
-    return genos_transpose, scores_after_c_t
+    with tempfile.NamedTemporaryFile(delete=False, suffix=".feather") as tmpfile:
```

Comment:
Can these matrix chunks be large? If so, could we either use the folder that the dosage matrix came from (can make a tmp dir in that folder), or thread through a user-provided temp_dir (which in the case of the cluster version, will use say the hg38.yml `temp_dir`, and default to the dosage matrix's folder if no temp_dir specified).

The reason is that the dosage matrix can be large, and currently we don't require that the worker nodes have large drives.